bicep/modules/kv_access_policy.bicep

```diff
@@ -17,5 +17,7 @@ resource kvAccessPolicy 'Microsoft.KeyVault/vaults/accessPolicies@2022-07-01' =
         permissions: permissions
       }
     ]
+    enableSoftDelete: true
+    enablePurgeProtection: true
```

Comment:
I do not see those properties in `KeyVault/vaults/accessPolicies` resource. The ones defined in `keyvault.bicep` should be enough.